public/locales/fr/translation.json

```diff
@@ -660,7 +659,8 @@
         "afterOrEqual": "Le champ {{field}} doit être postérieur ou égal au champ {{constraint}}",
         "beforeOrEqual": "Le champ {{field}} doit être antérieur ou égal au champ {{constraint}}",
         "equalDate": "Le champ {{field}} doit être égal au champ {{constraint}}",
-        "differentDate": "Le champ {{field}} doit être différent du champ {{constraint}}"
+        "differentDate": "Le champ {{field}} doit être différent du champ {{constraint}}",
+        "fileFormat": "Format de fichier invalide"
```

Comment:
The problem is that the check is not based on extensions but on MIME types and it will be not very understanding for users to have directly a list MIME types. So, provide a list of extensions correctly associated with validate MIME types is actually not easy doable.